gnomad_qc/v4/annotations/vrs_annotation_batch_v4.py

```diff
@@ -0,0 +1,520 @@
+
+        for vcf_name in file_list:
+            # Setting up jobs
+            new_job = init_job_with_gcloud(
+                batch=batch_vrs,
+                name=f"VCF_job_{vcf_name.split('.')[0].split('-')[1]}",
+                image=args.image,
+                mount=args.seqrepo_mount,
+                disk_size=args.disk_size,
+                memory=args.memory,
+            )
+
+            # Script path for annotation is not expected to change for GA4GH if
+            # dependencies are installed using pip in the DockerFile
+            vrs_script_path = "/usr/local/lib/python3.9/dist-packages/ga4gh/vrs/extras/vcf_annotation.py"
+
+            # Store VCF input and output paths
+            vcf_input = f"gs://{working_bucket}/vrs-temp/shards/shard-{version}.vcf.bgz/{vcf_name}"
+            vcf_output = f'/temp-vcf-annotated/annotated-{vcf_name.split(".")[0]}.vcf'
+
+            # Perform VRS annotation on vcf_input and store output in /vrs-temp
+            new_job.command("mkdir /temp-vcf-annotated/")
+            new_job.command(
+                f"python3 {vrs_script_path} --vcf_in"
+                f" {batch_vrs.read_input(vcf_input)} --vcf_out"
+                f" {vcf_output} --seqrepo_root_dir {seqrepo_path} --assembly"
+                f" {assembly} --vrs_attributes"
+            )
+
+            # bgzip the annotated VCF
+            new_job.command(f"bgzip {vcf_output}")
+
+            # Copy annotated shard to its appropriate place in Google Bucket
+            new_job.command(
+                "gsutil cp"
+                f" {vcf_output}.gz"
+                f" gs://{working_bucket}/vrs-temp/annotated-shards/annotated-{version}.vcf/"
+            )
+
+        # Execute all jobs in the batch_vrs Batch
+        batch_vrs.run()
+        logger.info(
+            "Batch jobs executed, annotated vcf.gz shards copied to"
+            f" gs://{working_bucket}/vrs-temp/annotated-shards/annotated-{version}.vcf"
+        )
+
+    if args.annotate_original:
+        logger.info(
+            "Batch jobs executed, preparing to read in sharded VCF from prior step."
+            " Preparing list of files first using Hail's hadoop_ls method."
+        )
+
+        annotated_file_dict = hl.utils.hadoop_ls(
+            f"gs://{working_bucket}/vrs-temp/annotated-shards/annotated-{version}.vcf/"
+        )
+        # NOTE: this is changed to be a directory intstead `/*.vcf` because the
+        # QoS step was run with Hail 0.2.115 instead of the latest version
+
+        annotated_file_list = [
+            annotated_file_item["path"] for annotated_file_item in annotated_file_dict
+        ]
+
+        logger.info("File list created. Now reading in annotated shards.")
+
+        # Import all annotated shards
+        ht_annotated = hl.import_vcf(
+            annotated_file_list,
+            force_bgz=True,
+            reference_genome=assembly,
+        ).make_table()
```

Comment:
I think this can be:
```suggestion
        ).rows()
```